web/src/test/java/org/springframework/security/web/authentication/www/BasicAuthenticationConverterTests.java

```diff
@@ -111,4 +111,15 @@ public void convertWhenEmptyPassword() {
 		assertThat(authentication.getName()).isEqualTo("rod");
 		assertThat(authentication.getCredentials()).isEqualTo("");
 	}
+
+	@Test
+	public void testWhenEmptyBasicAuthorizationHeaderTokenThenIgnored() {
```

Comment:
Although this test name follows the same pattern as other tests in this class, we would like any new tests to follow the convention methodWhenConditionThenResult.
A possible name could be:
```suggestion
	public void requestWhenEmptyBasicAuthorizationHeaderTokenThenIgnored() {
```